frontend/src/components/pages/your_clubs/ClubList.tsx

```diff
@@ -120,14 +109,22 @@ export function ClubList(props: ClubListProps) {
                 <div className={classes.clubHeader}>
                   <h2 className={classes.clubTitle}>{item.name}</h2>
                   <div className={classes.buttonContainer}>
-                    <Button
-                      className={classes.button}
-                      color="primary"
-                      endIcon={<PageviewIcon />}
-                      variant="contained"
+                    <Link to={{
+                      pathname:'/Club/' + item.name,
+                      state: {club: item}
```

Comment:
Should these be indented one more layer to show they're part of an object that is a prop?